### DISCUSSION TITLE
YOLOX calculate recall and precision

### DISCUSSION THREAD

**Discussion by 18369674616** [Megvii-BaseDetection/YOLOX#1215] [General]
I use VOC dataset for training. I'm doing a comparative experiment. I want to know how to see precision and recall. Now I can only see map. Other training models can be seen, such as faster R-CNN, yolov5, etc

**Comment by xxa783** [comment]
兄弟你解决了么。我也想这个，想看train_loss/ val_loss这些指标

**Reply by serosh1** [reply]
any idea, i need to get these too
